tests/DCPS/Presentation/main.cpp

```diff
@@ -223,7 +320,8 @@ ACE_TMAIN(int argc, ACE_TCHAR** argv)
 
       for (size_t i = 0; i < SAMPLES_PER_TEST; ++i)
       {
-        const Foo foo = {0, 0, 0, 0};
+        Foo foo {0, 0, 0, 0};
```

Comment:
This line is failing to compile on the CI builds that don't have C++11 enabled.

Reply:
Ok, thanks. It was unintentional to leave the equal sign out. I corrected it, and pushed the update. The build now fails on some timeout:
_"The job exceeded the maximum time limit for jobs, and has been terminated."_